.devcontainer/compose.yaml

```diff
@@ -2,13 +2,15 @@ name: "rails8_devcontainer_enhancements"
 
 services:
   rails-app:
+    image: rails8_devcontainer_enhancements_rails_app
```

Comment:
Does not work 100% as expected, see https://github.com/microsoft/vscode-remote-release/issues/2485 and https://github.com/devcontainers/cli/issues/306. But, at least I can reuse the same images and volumes.